aggregator-custom-query/query-aggregator.sh

```diff
@@ -50,7 +50,7 @@ if ! kubectl get namespace "$NAMESPACE" &>/dev/null; then
 fi
 
 # Find the aggregator pod
-POD_NAME=$(kubectl get pod -n "$NAMESPACE" -l app=aggregator -o jsonpath='{.items[0].metadata.name}')
+POD_NAME=$(kubectl get pods -n "$NAMESPACE" -o json | jq -r ".items[] | select(.spec.containers[].name == \"aggregator\") | .metadata.name")
```

Comment:
Would it be helpful to check and make sure the `jq` and `kubectl` binaries are installed at the beginning of this script?